server/src/rooms/game/state/index.ts

```diff
@@ -1120,9 +1120,11 @@ export class GameState extends Schema implements GameData {
     event.apply(this);
   }
 
+  // REFACTOR: DEFAULT CATEGORY
+  
   log(
     message: string,
-    category: string = 'Mars Event',
+    category: string | string = 'Mars Event',
```

Comment:
is this type declaration redundant?

Reply:
resolved in newest commit (37cbda3967fc514284dc76d91ff7ff8baeee9dd6)